common.sh

```diff
@@ -413,6 +413,20 @@ if [[ ! -z ${AGENT_E2E_TEST_SCENARIO} ]]; then
   fi
 
   case "$SCENARIO" in
+      "5CONTROL" )
+          export NUM_MASTERS=5
+          export MASTER_VCPU=4
+          export MASTER_DISK=100
+          export MASTER_MEMORY=32768
```

Comment:
I'm wondering if it could be better to have a 16G for the 4/5 cases, considering also the possibility to add some workers later - to not consume too much mem for the VMs

Reply:
I used 16G to test the 5 case and it worked fine. Let's try that.